src/extension.ts

```diff
@@ -135,6 +135,21 @@ export async function activate(context: vscode.ExtensionContext) {
 	// Implements the `RooCodeAPI` interface.
 	const socketPath = process.env.ROO_CODE_IPC_SOCKET_PATH
 	const enableLogging = typeof socketPath === "string"
+
+	// Watch the core files and automatically reload the extension host
+	const enableCoreAutoReload = process.env?.NODE_ENV === "development"
+	if (enableCoreAutoReload) {
+		console.log(`♻️♻️♻️ Core auto-reloading is ENABLED!`)
+		const watcher = vscode.workspace.createFileSystemWatcher(
+			new vscode.RelativePattern(context.extensionPath, "src/**/*.ts"),
```

Comment:
What happens if we change.a different file; e.g. one of the translation .json files?